code/injections/storage/dummy/stored_countries.go

```diff
@@ -182,7 +182,168 @@ var (
+			},
+			{
+				ImageURL:          util.StrPtr("https://geohints.com/Sources/Bollards/bollard_241.jpg"),
+				GoogleMapsExample: util.StrPtr("https://maps.google.com/maps?q=&layer=c&cbll=-45.9822522,168.7499536"),
+				Tags:              []string{"green", "white", "bollard", "mile marker"},
+			},
+		},
+		Signs: &core.Signs{
+			Yield: []core.Hint{
+				{
+					ImageURL: util.StrPtr("https://upload.wikimedia.org/wikipedia/commons/5/55/New_Zealand_road_sign_R2-2.svg"),
+				},
+				{
+					ImageURL: util.StrPtr("https://upload.wikimedia.org/wikipedia/commons/thumb/7/7a/New_Zealand_road_sign_R2-2F.svg/675px-New_Zealand_road_sign_R2-2F.svg.png"),
+				},
+			},
+			Stop: []core.Hint{
+				{
+					ImageURL: util.StrPtr("https://upload.wikimedia.org/wikipedia/commons/thumb/2/25/New_Zealand_road_sign_R2-1.svg/600px-New_Zealand_road_sign_R2-1.svg.png"),
+				},
+				{
+					ImageURL:    util.StrPtr("https://upload.wikimedia.org/wikipedia/commons/thumb/5/56/New_Zealand_road_sign_R2-4.svg/600px-New_Zealand_road_sign_R2-4.svg.png"),
+					Description: util.StrPtr("Stop sign for school areas"),
+				},
+				{
+					ImageURL:    util.StrPtr("https://upload.wikimedia.org/wikipedia/commons/thumb/d/d4/New_Zealand_road_sign_R2-4_%28perforated%29.svg/600px-New_Zealand_road_sign_R2-4_%28perforated%29.svg.png"),
+					Description: util.StrPtr("Alternative perforated top sign for school areas"),
+				},
+			},
+			Pedestrian: []core.Hint{
+				{
+					ImageURL:    util.StrPtr("https://upload.wikimedia.org/wikipedia/commons/thumb/8/86/New_Zealand_road_sign_W16-1-FYG.svg/1024px-New_Zealand_road_sign_W16-1-FYG.svg.png"),
+					Description: util.StrPtr("Standard pedestrian crossing sign"),
+				},
+				{
+					ImageURL:    util.StrPtr("https://upload.wikimedia.org/wikipedia/commons/thumb/c/cf/New_Zealand_road_sign_W16-2-FYG.svg/600px-New_Zealand_road_sign_W16-2-FYG.svg.png"),
+					Description: util.StrPtr("Cross walk coming up sign"),
+				},
+			},
+			Speed: []core.Hint{
+				{
+					ImageURL:    util.StrPtr("https://upload.wikimedia.org/wikipedia/commons/thumb/4/4a/New_Zealand_road_sign_R1-1_%2860%29.svg/1024px-New_Zealand_road_sign_R1-1_%2860%29.svg.png"),
+					Description: util.StrPtr("Standard speed sign"),
+				},
+				{
+					ImageURL:    util.StrPtr("https://upload.wikimedia.org/wikipedia/commons/thumb/7/7d/New_Zealand_road_sign_R1-2_%2860%29.svg/1024px-New_Zealand_road_sign_R1-2_%2860%29.svg.png"),
+					Description: util.StrPtr("Variable speed sign. Can also have black background"),
+				},
+				{
+					ImageURL:           util.StrPtr("https://upload.wikimedia.org/wikipedia/commons/9/99/Products-ez-szsk144-rs62-w16-5-1-kura-school-speed-30km-sign-thumb.jpg"),
+					Description:        util.StrPtr("Maori Te Kura school zone sign"),
+					Significance:       util.StrPtr("There is only one Te Kura school in the South of NZ. The rest are in the North."),
+					RegionSpecificity:  util.Float32Prt{0.4},
+					CountrySpecificity: util.Float32Prt{1},
```

Comment:
Does....does this syntax work? normally I see this for instantiating a struct, not making a function call